src/shared/components/query/filteredSearch/Phrase.tsx

```diff
@@ -165,7 +165,8 @@ function matchPhrase(phrase: string, fullText: string) {
 
 /**
  * Full match using lowercase
+ * Need to convert boolean to string before applying lowercase
  */
-function matchPhraseFull(phrase: string, fullText: string) {
-    return fullText.toLowerCase() === phrase.toLowerCase();
+function matchPhraseFull(phrase: string, toMatch: boolean | string | number) {
+    return _.toString(toMatch).toLowerCase() === phrase.toLowerCase();
```

Comment:
probably don'tneed to use _ (lodash here).  just toMatch.toString().toLowerCase()